lib/Service/RoomService.php

```diff
@@ -132,7 +132,7 @@ public function createOneToOneConversation(IUser $actor, IUser $targetUser): Roo
 	 */
 	public function createConversation(int $type, string $name, ?IUser $owner = null, string $objectType = '', string $objectId = ''): Room {
 		$name = trim($name);
-		if ($name === '' || isset($name[255])) {
+		if ($name === '' || strlen($name) > 255) {
```

Comment:
This russian string is "167" chars long, but it is not allowed:
```
орем ипсум долор сит амет, агам аргументум еффициантур ан яуо. Стет уллум при ех, ад ребум фацер сцрипта яуо, вим алтерум адмодум праесент ад. Хис омнес персецути ат.
```

```suggestion
		if ($name === '' || mb_strlen($name) > 255) {
```

Would fix it.

Reply:
The size of database column is 255

Reply:
To store Russian characters we will need a more big column, not?!

Reply:
No, because the DB sees them as mb chars, using the mb_strlen will store it correctly in the database.
Just try it :)

Reply:
Cool, in Portugese we don't write with unicode chars. Worked fine with Russian chars.

```
php > echo mb_strlen(str_repeat('и', 255));
255
```